### PR TITLE
added support for docker log drivers

### PR DESCRIPTION
**Pull request by madushan1000**

### CODE REVIEW

**Comment by madushan1000** [comment]
@artpolikarpov's PR (https://github.com/arunoda/meteor-up/pull/788) on old mupx was very useful here.

**Comment by artpolikarpov** [comment]
✌️

File: tests/test-run.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ function run_test {
     export PROD_SERVER=$(sudo docker inspect --format '{{ .NetworkSettings.IPAddress }}' $DOCKER_ID)
     cd ~/meteor-up
     npm test -- -g $1
-    sudo docker rm -f $DOCKER_ID > /dev/null
+    sudo docker rm -f $(sudo docker stop $DOCKER_ID) > /dev/null
```
Comment (arunoda) [inline tests/test-run.sh]: Could you write a test case which

* configure the app to use syslog
* then try to check syslog and see whether we are getting logs or not.

**Comment by arunoda** [comment]
Thanks @artpolikarpov for finding the issue in the first place.

**Comment by madushan1000** [comment]
updated with the test

**Pull request merged**